src/Authenticator/JwtAuthenticator.php

```diff
@@ -166,6 +168,27 @@ protected function decodeToken(string $token): ?object
             );
         }
 
+        $jsonWebKeySet = $this->getConfig('jsonWebKeySet');
+        if ($jsonWebKeySet) {
+            $keySet = JWK::parseKeySet($jsonWebKeySet);
+            /*
+             * TODO Converting Keys to Key Objects is no longer needed in firebase/php-jwt ^6.0
+             * @link https://github.com/firebase/php-jwt/pull/376/files#diff-374f5998b3c572d86be0e79432aac3de362c79e8fb146b9ce422dc2388cdc5daR50
+             */
+            $keyAlgorithms = array_combine(
```

Comment:
Using the `Hash` class to generate the `Key` objects array will lead to more readable code.